symbol-processing-aa-embeddable/build.gradle.kts

```diff
@@ -203,3 +203,32 @@ signing {
     useInMemoryPgpKeys(signingKey, signingPassword)
     sign(extensions.getByType<PublishingExtension>().publications)
 }
+
+abstract class WriteVersionSrcTask @Inject constructor(
+    @get:Input val kspVersion: String,
```

Comment:
seems that kspVersion is not used other than being annotated with `Input` to potentially trigger rebuild?

Reply:
updated!